src/database/migrations/20200213061003-create-room.js

```diff
@@ -6,18 +6,45 @@ export default {
       primaryKey: true,
       type: Sequelize.INTEGER
     },
-    type: {
```

Comment:
We need the room type too. eg. Dulexe rooms, Standard rooms and they all have different costs.

Reply:
resolved